change/@fluentui-web-components-2021-01-04-11-29-21-users-v-sedono-fix-inline-start-margin-rtl-slider.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "add inline start margin for rtl to slider",
```

I think this may benefit by being a bit more precise about the purpose of the fix:
```suggestion
  "comment": "fix vertical slider track margin in RTL scenarios",
  ```